documentation/projects/proposals/nuxt_3_migration/20230608-implementation_plan_nuxt_3_migration.md

```diff
@@ -0,0 +1,338 @@
+| `"@nuxtjs/redirect-module"`        | Define Nitro routeRules in `nuxt.config`                                                                | https://nitro.unjs.io/config/#routerules                                                                                                                 |
+| `"@nuxtjs/sentry"`                 | No official support                                                                                     | Workaround in https://github.com/nuxt-community/sentry-module/issues/530                                                                                 |
+| `"vue-plausible"`                  | ~~Update to current version that supports Nuxt 3 or~~ Move to https://github.com/nuxt-modules/plausible | Note: enabling outbound tracking breaks links with target blank in `vue-plausible`                                                                       |
+| `"@nuxtjs/sitemap"`                | Replace with `nuxt-simple-sitemap`                                                                      | https://github.com/harlan-zw/nuxt-simple-sitemap                                                                                                         |
+
+We should probably add [`nuxt-vitest`](https://github.com/danielroe/nuxt-vitest)
+for testing in Nuxt. The package is under active development, so we should pin
+to the patch version
+
+## Alternatives
+
+<!-- Describe any alternatives considered and why they were not chosen or recommended. -->
+
+Migrate to a non-Nuxt SSR solution for Vue. This would require significant
+changes to code for an unclear benefit.
+
+## Design
+
+<!-- Note any design requirements for this plan. -->
+
+The designs should stay exactly the same. Hopefully, all of the Playwright
+visual regression tests pass after this migration.
+
+## Parallelizable streams
+
+<!-- What, if any, work within this plan can be parallelized? -->
+
+The Nuxt 2 changes can be worked on immediately. The changes to Nuxt 3 branch
+can be parallellized after the initial PR.
+
+## Blockers
+
+<!-- What hard blockers exist which might prevent further work on this project? -->
+
+## Accessibility
+
+<!-- Are there specific accessibility concerns relevant to this plan? Do you expect new UI elements that would need particular care to ensure they're implemented in an accessible way? Consider also low-spec device and slow internet accessibility, if relevant. -->
+
+## Rollback
+
+<!-- How do we roll back this solution in the event of failure? Are there any steps that can not easily be rolled back? -->
+
+Since Nuxt 3 specific features will be in a separate branch, if there is a need
+to roll back after merging `nuxt3` branch, we can revert the merge commit.
+
+## Risks
```

Comment:
Some other risks I can think of:

1. Drift between the `nuxt3` and `main` branches, or difficulty keeping them in sync
2. Encountering an unexpected dependency incompatability that halts the migration, further exacerbating pt.1